backend/remote/backend_state.go

```diff
@@ -31,12 +32,12 @@ func (r *remoteClient) Get() (*remote.Payload, error) {
 			// If no state exists, then return nil.
 			return nil, nil
 		}
-		return nil, fmt.Errorf("Error retrieving remote state: %v", err)
+		return nil, fmt.Errorf("Error retrieving  state: %v", err)
```

Comment:
Nit: fyi you have an extra space here left where you removed `remote`

Reply:
Good catch! Let me update that!

Reply:
Updated...